src/microbe_stage/MicrobeColony.cs

```diff
@@ -45,6 +49,31 @@ public Microbe.MicrobeState State
     [JsonProperty]
     public Microbe Master { get; private set; }
 
+    /// <summary>
+    ///   The total hex count from all members of this colony.
+    /// </summary>
+    [JsonIgnore]
+    public float HexCount
+    {
+        get
+        {
+            if (hexCountDirty)
+                UpdateHexCount();
+            return hexCount;
+        }
+    }
+
+    [JsonIgnore]
+    public float IngestedSizeCount
```

Comment:
What is "IngestedSize"? This variable is not clear at all to me, is it the currently ingested amount of stuff? Or the limit of how much ingested stuff there can be at once. This should be renamed to something a bit more specific (and the related property in Microbe)

Reply:
I can't find any good words for a microbe full-o-meter, maybe `Fullness`?

Reply:
Is it in range 0-1 or in 1-n (where n is the max number of things to engulf)? If it is a fraction, then I'd name this "UsedIngestionCapacity" or "IngestedObjectCount" / "IngestedEntityCount" for the latter case.

Reply:
It is not fraction, it's the accumulation of all member's `Microbe.EngulfSize`. Will rename this to UsedIngestionCapacity regardless and improve docs.

Reply:
Sounds good.